config/client.go

```diff
@@ -14,7 +14,7 @@ type ClientConfig struct {
 
 func ParseClientConfig() (cfg ClientConfig) {
 	flag.StringVar(&cfg.ChainURL, "chain-url", "ws://127.0.0.1:8545", "'protocol://ip:port'")
-	flag.StringVar(&cfg.Mnemonic, "mnemonic", "pistol kiwi shrug future ozone ostrich match remove crucial oblige cream critic", "Wallet mnemonic.")
+	flag.StringVar(&cfg.Mnemonic, "mnemonic", "tag volcano eight thank tide danger coast health above argue embrace heavy", "Wallet mnemonic.")
```

Comment:
why did you change the mnemonic from the readme tutorial?

Reply:
Hm i changed it to the default mnemonic in the operator unaware that we also used it in the `README.md`. I will instread change the default mnemonic of the Operator.

Reply:
I just understood that we have two `README` files with different mnemonics :see_no_evil: i will change them to the same.